site/install-cluster-operator.md

```diff
@@ -0,0 +1,245 @@
+# Installing Cluster Operator in a Kubernetes cluster
+
+## <a id="overview" class="anchor" href="#overview">Overview</a>
+
+This guide covers the installation of Cluster Operator in a Kubernetes cluster. This guide
+was written using a GKE cluster, although any Kubernetes flavour should work.
+
+There are two ways to install the Cluster Operator:
+
+ * Using a Helm chart
+ * Using `kubectl` and YAML manifests
+
+Regardless of the installation method chosen, download or clone the
+[Cluster Operator repository](https://github.com/rabbitmq/cluster-operator) and set it as
+your current working directory.
+
+## Compatibility
+
+- Kubernetes 1.16 or 1.17
+- Kubernetes 1.18 is not fully tested and there might be incompatibilities
+- [RabbitMQ DockerHub image](https://hub.docker.com/_/rabbitmq) 3.8.5+
```

Comment:
We refer to the image as "Community RabbitMQ Docker image" to emphasize that it's not maintained by the core team or VMware in general.